src/chi/components/avatars/avatars.scss

```diff
@@ -12,7 +12,7 @@ $avatar-sizes: (
   lg: 4rem,
   xl: 6rem,
   xxl: 8rem);
-$avatar-colors: (grey, red, pink, purple, indigo, navy, blue, teal, mint, green, yellow, orange, inverse, secondary, primary);
+$avatar-colors: (grey, red, pink, purple, indigo, navy, blue, teal, mint, green, yellow, orange, light, secondary, primary);
```

Comment:
@davidojedacl we shouldn't remove the inverse color from avatars, it should stay as legacy code, the same that we did with buttons.

Reply:
done